code.gs

```diff
@@ -1845,6 +1845,37 @@ var i18n = {
     'It looks like you are using an outdated version of this script': 'Atrodo, kad jūs naudojate pasenusią šio skripto versiją',
     'You can find the latest one here': 'Naujausią galite rasti čia',
   },
+  'he': {
+    'Age': 'גיל',
+    'Years': 'שנים',
+    'Events': 'אירועים',
+    'Birthdays today': 'ימי הולדת היום',
+    'Birthdays tomorrow': 'ימי הולדת מחר',
+    'Birthdays in {0} days': 'ימי הולדת ב-{0} הימים הקרובים',
```

Comment:
I do not know hebrew at all, so I just threw your translation into Google Translator to check for any major issues or misunderstadings.  
There is only one thing I've found which we should verify before proceeding.

This string (along with the other two similar ones at lin 1857 and 1860) is being translated as "Birthdays in the next {0} days", but the meaning of "in X days" in the original string is not quite the same of "in the next X days".  
Is this an error of the translator? If not I'd ask you to change your translation to keep the original meaning.

Thanks

Reply:
I'll rephrase to make it clearer :+1: